### PR TITLE
Player new Method AddAhp

### PR DESCRIPTION
**Pull request by louis1706**
Making easy to make Custom AHP

### CODE REVIEW

File: Exiled.API/Features/Player.cs [hunk 1]
```diff
@@ -2037,6 +2037,20 @@ public void ChangeEffectIntensity(EffectType type, byte intensity, float duratio
         /// <returns>The tantrum's <see cref="GameObject"/>.</returns>
         public GameObject PlaceTantrum() => Map.PlaceTantrum(Position);
 
+        /// <summary>
+        /// Give an new AhpProccess.
```
Comment (joker-119) [inline Exiled.API/Features/Player.cs:2041]: "Gives a new <see cref="AhpStat">."

File: Exiled.API/Features/Player.cs [hunk 2]
```diff
@@ -2037,6 +2037,20 @@ public void ChangeEffectIntensity(EffectType type, byte intensity, float duratio
...
+        /// </summary>
+        /// <param name="amount">The ammount of AHP he get.</param>
```
Comment (joker-119) [inline Exiled.API/Features/Player.cs:2043]: The amount to give the player.

File: Exiled.API/Features/Player.cs [hunk 3]
```diff
@@ -2037,6 +2037,20 @@ public void ChangeEffectIntensity(EffectType type, byte intensity, float duratio
...
+        /// <param name="limit">The maximum AHP he can get.</param>
```
Comment (joker-119) [inline Exiled.API/Features/Player.cs:2044]: The maximum AHP for this stat.

File: Exiled.API/Features/Player.cs [hunk 4]
```diff
@@ -2037,6 +2037,20 @@ public void ChangeEffectIntensity(EffectType type, byte intensity, float duratio
...
+        /// <param name="decay">How many it's lost per tick.</param>
```
Comment (joker-119) [inline Exiled.API/Features/Player.cs:2045]: How much value is lost per second.

File: Exiled.API/Features/Player.cs [hunk 5]
```diff
@@ -2037,6 +2037,20 @@ public void ChangeEffectIntensity(EffectType type, byte intensity, float duratio
...
+        /// <param name="efficacy">percentage of damage it's take on Hurt.</param>
```
Comment (joker-119) [inline Exiled.API/Features/Player.cs:2046]: Percent of incoming damage absorbed by this stat.

File: Exiled.API/Features/Player.cs [hunk 6]
```diff
@@ -2037,6 +2037,20 @@ public void ChangeEffectIntensity(EffectType type, byte intensity, float duratio
...
+        /// <param name="sustain">Time before the AHP is decay.</param>
```
Comment (joker-119) [inline Exiled.API/Features/Player.cs:2047]: The number of seconds to delay the start of the decay.

File: Exiled.API/Features/Player.cs [hunk 7]
```diff
@@ -2037,6 +2037,20 @@ public void ChangeEffectIntensity(EffectType type, byte intensity, float duratio
...
+        /// <param name="persistant">The Proccess it's not deleted when ahp it's at 0.</param>
```
Comment (joker-119) [inline Exiled.API/Features/Player.cs:2048]: Whether or not the process is removed when the value hits 0.

**Review by joker-119 — CHANGES_REQUESTED** [review]

File: Exiled.API/Features/Player.cs [hunk 8]
```diff
@@ -2038,14 +2038,14 @@ public void ChangeEffectIntensity(EffectType type, byte intensity, float duratio
...
         /// <summary>
-        /// Give an new AhpProccess.
-        /// </summary>
-        /// <param name="amount">The ammount of AHP he get.</param>
-        /// <param name="limit">The maximum AHP he can get.</param>
-        /// <param name="decay">How many it's lost per tick.</param>
-        /// <param name="efficacy">percentage of damage it's take on Hurt.</param>
-        /// <param name="sustain">Time before the AHP is decay.</param>
-        /// <param name="persistant">The Proccess it's not deleted when ahp it's at 0.</param>
+        /// Gives a new.
```
Comment (joker-119) [inline Exiled.API/Features/Player.cs:2041]: Gives the player a new AHP Process.

**Review by joker-119 — CHANGES_REQUESTED** [review]

File: Exiled.API/Features/Player.cs (hunk 8 above)
Comment (louis1706) [reply Exiled.API/Features/Player.cs:2041]: if i put 
/// Gives a new <see cref="AhpStat">adding ahp</see>.
is it good ?

Comment (joker-119) [reply Exiled.API/Features/Player.cs:2041]: No, that's also not proper english.
Just put what I said.

Comment (louis1706) [reply Exiled.API/Features/Player.cs:2041]: Okay

**Review by joker-119 — APPROVED** [review]

**Pull request merged**